Sources/BSON/Document.swift

```diff
@@ -28,7 +87,20 @@ public struct Document {
      *
      * - SeeAlso: http://bsonspec.org/
      */
-    public init(fromBSON bson: Data, validate: Bool = true) throws { fatalError("Unimplemented") }
+    public init(fromBSON bson: Data, validate: Bool = true) throws {
```

Comment:
we'll need to update this based on the rebase. I'm fine with deferring validation to a later PR, but let's create a ticket and add a TODO so we remember to add it.

Reply:
https://jira.mongodb.org/browse/SWIFT-866
adding the todo in the next commit